src/modules/dashboard/components/ManageFundsDialog/ManageFundsDialog.tsx

```diff
@@ -155,6 +170,19 @@ const ManageFundsDialog = ({
       icon: 'emoji-medal',
       comingSoon: true,
     },
+    {
+      title: MSG.unlockTokensTitle,
+      description: MSG.unlockTokensDescription,
+      icon: 'emoji-padlock',
```

Comment:
You forgot to commit the actual emoji file :)

![Screenshot from 2021-02-12 17-12-08](https://user-images.githubusercontent.com/1193222/107786078-101ec880-6d56-11eb-876f-8e572f8ee495.png)

Reply:
Lost it while rebasing 🤦  Added